docs/application/index.md

```diff
@@ -1,178 +1,323 @@
+	}
+
+	#wrapper #main .docs-content-boxes-brand-boxes .heading-with-icon p {
+	    font-size: 1.4rem !important;
+		height: 95;
+		min-height: unset;
+	}
+	.docs-content-boxes.docs-content-boxes-brand-boxes {
+		flex-direction: column;
+		margin-left: 10px;
+		margin-right: 10px;
+		margin-bottom: 60px;
+	}
+	.docs-content-boxes.docs-content-boxes-brand-boxes .column {
+		flex: 0 0 auto;
+		width: 378px;
+		margin: 10px auto;
+	}
+	
+	#wrapper {
+	   overflow: auto !important;
+  }
+}
 
--   [Native application](native/index.md) is developed using C and can access more advanced device-specific features, such as camera, GPS, and accelerometer in addition to more advanced system settings and functionality.
+@media only screen and (max-width:640px) {
+	.docs-column-wrapper {
+		background-color: transparent !important;
+	}
 
-    The native applications use the Native API, which provides all of the memory management and performance benefits that come with building applications for Linux in C. The Native API is extremely helpful, as it includes dozens of API modules that cover a large range of capabilities. It provides numerous interfaces to much of the hardware that is found in modern mobile and wearable devices, and does so in an environment that is tailored for limited resources.
+	#wrapper {
+		overflow: auto !important;
+	}
+}
+
+@media only screen and (max-width: 600px) {
+	.finalcta.home {
+		margin: 0 0 !important;
+	}
+}
+
+@media screen and (max-width: 500px) {
+	.docs-content-boxes.docs-content-boxes-brand-boxes .column {
+		max-width: 350px;
+		width: auto;
+	}
+}
 
-The following figure illustrates the Tizen architecture model supporting the 3 application types.
+.docs-content-boxes.content-boxes-icon-on-side .heading-with-icon, 
+.docs-content-boxes.content-boxes-icon-with-title .heading-with-icon {
+	display: table;
+	width: auto;
+}
 
-**Figure: Tizen architecture**
+.docs-content-boxes.docs-content-boxes-brand-boxes .content-box-wrapper:hover {
+	box-shadow: 0px 1.2px 3.6px rgba(0, 0, 0, 0.108), 0px 6.4px 14.4px rgba(0, 0, 0, 0.132);
+	transition-duration: 0.3s;
+}
 
-![Tizen architecture](media/what_is_tizen_architecture.png)
+</style>
 
-The Tizen platform also allows you to develop a hybrid application package where native and Web applications are packaged together to make more powerful applications. The Tizen platform ensures that all Tizen applications have consistent look and feel, regardless of whether you use the native or Web framework to create them.
+<section id ="main">
+Tizen is a user-interactive and service-oriented open source project that allows you to create feature-rich applications for multiple device categories.
 
+To get started with the development of your own Tizen applications:
 
+<div class="docs-ui-get-started">
+	<div class="docs-content-boxes content-boxes-1 content-boxes-icon-with-title docs-content-boxes-brand-boxes" data-animationoffset="100%" style="margin-top:10px;margin-bottom:10px;">
```

Comment:
Suggestion:
The layout appearance is fine overall but could we please reduce the extra white spaces for the wearable, TV, and Mobile boxes under Get Started heading.

I understand it is to maintain the page width with the next section but we could align the image better.